lib/ui/home/home_view_model.dart

```diff
@@ -11,11 +14,30 @@ final homeViewModelProvider =
               getIt.get<GetProfileUseCase>(),
             ));
 
+final todayStream = StreamProvider.autoDispose<String>((ref) =>
+    ref.watch(homeViewModelProvider.notifier)._todayStreamController.stream);
+
+final profileStream = StreamProvider.autoDispose<ProfileModel>((ref) =>
+    ref.watch(homeViewModelProvider.notifier)._profileStreamController.stream);
+
+final visibleIndexStream = StreamProvider.autoDispose<int>((ref) => ref
+    .watch(homeViewModelProvider.notifier)
+    ._visibleIndexStreamController
+    .stream);
+
 class HomeViewModel extends StateNotifier<HomeViewState> {
   final GetProfileUseCase _getProfileUseCase;
 
+  final _todayStreamController = StreamController<String>();
+  final _profileStreamController = StreamController<ProfileModel>();
+  final _visibleIndexStreamController = StreamController<int>();
```

Comment:
please add `dispose` for stream values in dispose function of ViewModel
![image](https://user-images.githubusercontent.com/60863885/230534817-e4718bc6-1f16-492b-a19c-292883b69344.png)